eventmesh-common/src/main/java/org/apache/eventmesh/common/config/CommonConfiguration.java

```diff
@@ -111,8 +112,8 @@ public class CommonConfiguration {
     @ConfigFiled(reload = true)
     private String meshGroup;
 
-    @ConfigFiled(field = "server.retry.storageEnabled")
-    private boolean eventMeshServerRetryStorageEnabled = Boolean.FALSE;
+    @ConfigFiled(field = "server.retry.plugin.type")
+    private String eventMeshRetryPluginType = Constants.STANDALONE;
```

Comment:
Isolation is configured to allowing users to retry using standalone mode when the storage medium is not using standalone.

---

是的, 配置隔离是为了应对当存储介质不使用standalone时, 也允许用户使用standalone模式重试。

Reply:
I understand that the original intention here is to use the `default` strategy. There is no need to reflect it as standalone. It can be marked as default. Others, such as rocketmq and rabbitmq, can have different retry strategies. `eventMeshRetryPluginType = “default”;` That’s fine. @pandaapo @yanrongzhen 

---
我理解这里的本意是走默认的策略，没必要体现是standalone，可以标识为default，其他例如rocketmq，rabbitmq，可以有不同的重试策略实现，`eventMeshRetryPluginType = “default”;` 这样就好。 @pandaapo @yanrongzhen

Reply:
The relationship between the retry strategy plugin and storage can be reflected in the document, and there is no need to strongly bind the configuration name and code logic. I think everyone can understand this.

---
重试策略插件与使用的存储的关系，可以体现在文档中，不用在配置名称和代码逻辑上强绑定。这点我想大家也是都能理解的。

Reply:
I understand that the current modification has nothing to do with `standalone`. If the `retry strategy` is configured, the corresponding configuration will be used. If the configuration is not loaded, the `default strategy` will be used. For the situation the rocketmq strategy is used, but the rabbitmq is used for storage. Just mark it in the documentation guide, or add a judgment to determine whether the strategy matches the storage, or throw a clear mismatch exception.

---
我理解目前的修改与`standalone` 没什么关系了，配置了`retry策略`就是走对应配置的，如果配置的没加载到就是走`default策略`，对于使用retry策略使用的`rocketmq策略`，但是存储使用的rabbitmq的这种情况，在文档指引中体现标识一下就行了，或者就是加一个策略与存储是否匹配的判断，抛出一个明确的不匹配的异常也可以。

Reply:
The judgment of whether it matches needs to determine whether the strings of the two plug-in types are equal, which may lead to the coupling of the configuration.

---

是否匹配的判断需要判断两者插件类型的字符串是否相等, 感觉可能会导致配置的耦合。

Reply:
Yes, there will be coupling, so either there are relevant descriptions in the document guidelines to make it clear to users, or coupling judgment processing must be carried out in the code. The judgment conditions are: if the retry strategy is used or the corresponding The storage module is the same, or it is the default strategy.

---
是的，是会存在耦合的情况，所以要么文档指引中有相关描述，让使用人清楚，要么就是要在代码中进行耦合的判断处理，判断的条件就是：如果使用了重试策略要么与对应的storage模块相同，要么就是default策略。